step.yml

```diff
@@ -9,16 +9,17 @@ description: |-
   
   Before you start using this Step, you need to do a couple of things:
   
-  * Register an app on the **My Apps** page of iTunes Connect. Click on the *plus sign* and select the **New App** option. This requires an **admin** account.
+  * Register an app on the **My Apps** page of App Store Connect. Click on the *plus sign* and select the **New App** option. This requires an **admin** account.
   * This Step requires an app signed with App Store Distibution provisioning profile. Make sure that you use the correct code signing files and the correct export method with the Step that builds your binary.
-  * Every build that you want to push to iTunes Connect must have a unique build and version number pair. Increment either or both before a new deploy to iTunes Connect.
+  * Every build that you want to push to App Store Connect must have a unique build and version number pair. Increment either or both before a new deploy to App Store Connect.
   
   To deploy your app with the Step: 
   1. Make sure that either the **IPA path** or the **PKG path** input has a valid value. The default value is perfect for most cases: it points to the output generated by the **Xcode Archive** Step.
-  1. In the **Platform** input, set the platform of your app. If, for example, you have an iOS app, set it to `ios`. The Step will fail if this input isn't set correctly.
-  1. Set up your connection depending on which authentication method you wish to use: with Apple ID or with the App Store Connet API key. Use only one of the authentication methods. 
-    - For the Apple ID: Fill out the **Apple ID**, **Password**. Optionally, add the **Application specific password** if you wish to bypass two-factor authentication on iTunesConnect.
-    - For the App Store Connect API: Provide your **API Key URL** (for example, https://URL/TO/AuthKey_something.p8 or file:///PATH/TO/AuthKey_something.p8) and the **Issuer ID** inputs. Please note that connecting with the App Store Connect API works with XCode 11.0 and above releases.  
+  1. Set up your connection depending on which authentication method you wish to use:
+      - Use a previously set up Bitrise Apple Developer connection, set the **Bitrise Apple Developer Connection** to `automatic` (this is the default setting), `api_key` or `apple_id`.
+      - Provide manual Step inputs: either with Apple ID or with the App Store Connet API key. Set the **Bitrise Apple Developer Connection** to `off`. Use only one of the authentication methods. 
```

Comment:
- Provide manual Step inputs: Add authentication data depending on which authentication method you wish to use, either Apple ID or API key authentication. Set the **Bitrise Apple Developer Connection** to `off`. Use only one of the authentication methods.